src/framework/mpas_particle_list_types.inc

```diff
@@ -7,7 +7,7 @@
        type (mpas_pool_type), pointer :: haloDataPool => NULL()
        ! pool containing particle data not needing to be passed
        ! between processors
-       type (mpas_pool_type), pointer :: nonhaloDataPool => NULL()
+       !type (mpas_pool_type), pointer :: nonhaloDataPool => NULL()
```

Comment:
It looks like this is a simple repeated like error? It would need a separate framework PR, but would be easy and fast to get through.

Reply:
I'm removing the `nonHalo` feature until it is needed.  But, you are right, it is a simple one-liner.  I'll remove it and just leave it in framework for now because it isn't a big deal.